tox.ini

```diff
@@ -0,0 +1,85 @@
+[tox]
+
+[testenv:all-deps]
+# skip install to avoid rebuilding the world, do the install with setup.py
+# instead of pip below
+skip_install = True
+passenv =
+    RASCALINE_BUILD_TYPE
+    RUSTUP_HOME
+    CARGO*
+
+deps =
+    discover
+    numpy
+    ase
+    chemfiles
+
+commands =
+    python setup.py --quiet install
+    discover -p "*.py" -s python/tests
+
+[testenv:min-deps]
+skip_install = True
+passenv =
+    RASCALINE_BUILD_TYPE
+    RUSTUP_HOME
+    CARGO*
+
+deps =
+    discover
+    numpy
+
+commands =
+    python setup.py --quiet install
+    discover -p "*.py" -s python/tests
+
+[testenv:examples]
+skip_install = True
+passenv =
+    RASCALINE_BUILD_TYPE
+    RUSTUP_HOME
+    CARGO*
+
+deps =
+    numpy
+    chemfiles
+    ase
+
+commands =
+    python setup.py --quiet install
+    python python/tests/check_examples.py
+
+[testenv:lint]
+deps =
+    flake8>=4
+    flake8-docstrings
+    flake8-bugbear
+    pygments
+    isort
+skip_install = true
+commands =
+    flake8 {posargs:python setup.py}
+    isort --verbose --check-only --diff src/mdacli tests setup.py
+
+[flake8]
+max_line_length = 80
```

Comment:
flake8 and isort should be configured with a line size of 88 since that's what black uses

Reply:
Fine for me!